src/cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.js

```diff
@@ -112,6 +113,19 @@ function createPaymentRequest(args) {
       args.addTaxPercentage = true;
       if(paymentRequest.paymentMethod.type.indexOf('klarna') > -1){
         args.addTaxPercentage = false;
+        const address = order.getBillingAddress();
+        const otherDeliveryAddress = {
+          shipping_method: order.getDefaultShipment().shippingMethod.displayName,
+          shipping_type: order.getDefaultShipment().shippingMethod.description,
```

Comment:
maybe we can define the `defaultShipment` variable same as address in lie 116 instead of invoking the service twice?